WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/logic/ui/listeners/WorkbenchListener.java

```diff
@@ -58,24 +58,29 @@ private void addShutdownListeners() {
 		workbench.addWorkbenchListener(new IWorkbenchListener() {
 
 			private IntervalInitializationManager intervalInitializationManager;
+			private EventInitializationManager eventInitializationManager;
 
 			@Override
 			public boolean preShutdown(final IWorkbench workbench,
 					final boolean forced) {
 				intervalInitializationManager = IntervalInitializationManager
 						.getInstance();
+				eventInitializationManager = EventInitializationManager
+						.getInstance();
 				eventManager.update(
 						new WatchDogEvent(workbench, EventType.END_IDE));
 				intervalInitializationManager.getIntervalManager()
 						.closeAllIntervals();
-				intervalTransferManager.sendIntervalsImmediately();
+				intervalTransferManager.sendItemsImmediately();
+				eventInitializationManager.getEventTransferManager()
+						.sendItemsImmediately();
 				return true;
 			}
 
 			@Override
 			public void postShutdown(final IWorkbench workbench) {
 				intervalInitializationManager.shutdown();
-				EventInitializationManager.getInstance().shutdown();
```

Comment:
No, it is the same, see the next line (line 83) and line 68/69 (as well as the fact that EventInitializationManager is a singleton class) ;)